domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClientStreamResult.java

```diff
@@ -0,0 +1,59 @@
+package org.folio.rest.persist;
+
+import io.vertx.core.Handler;
+import org.folio.rest.jaxrs.model.ResultInfo;
+
+class PostgresClientStreamResult<T> {
+
+  private final ResultInfo resultInfo;
+
+  private Handler<T> streamHandler;
+  private Handler<Void> endHandler;
+  private Handler<Throwable> exceptionHandler;
+
+  public PostgresClientStreamResult(ResultInfo resultInfo) {
+    this.resultInfo = resultInfo;
+  }
+
+  ResultInfo resultInto() {
+    return this.resultInfo;
+  }
+
+  public void handler(Handler<T> streamHandler) {
```

Comment:
Can you change `void` to `PostgresClientStreamResult` and return `this` for chaining calls (fluent API)? (all 3 setters)
Can you add javadoc? (all 3 setters)